src/index.js

```diff
@@ -1,5 +1,9 @@
 const app = require('./app');
+const db = require('./../database/db_connection.js');
 
-app.listen(app.get('port'), () => {
-  console.log(`App running on port ${app.get('port')}`);
+db.once('open', function () {
+  console.log('connected to DB');
+  app.listen(app.get('port'), () => {
+    console.log('Magic happens on port 8181!');
```

Comment:
the original one was a bit better, cause what if you wanted to change the port ???
also i would do 
```js 
process.env.PORT || app.get('port')
```
for heroku setup